templates/class.mustache

```diff
@@ -30,10 +30,15 @@ export interface ResponseWithBody<S extends number, T> extends Response {
     body: T;
 }
 
+export interface QueryParameters { [param: string]: any };
```

Comment:
nit: We typically use type declarations for this instead of interfaces, even though we have some inconsistencies